static/templates/user_info_popover_content.hbs

```diff
@@ -67,15 +67,15 @@
             </a>
         </li>
         {{/if}}
-        <li>
-            <a href="#" class="update_status_text">
-                <i class="fa fa-comments" aria-hidden="true"></i>
-                {{#if status_text}}
-                    {{#tr this}}Edit status message{{/tr}}
-                {{else}}
-                    {{#tr this}}Set a status message{{/tr}}
-                {{/if}}
-            </a>
+        <li class="update_status_text" id="status_messages">
+            <label for="user_status"><i class="fa fa-comments" aria-hidden="true"></i>Set status message</label>
+            <input type="text" class="user_status" maxlength="22" />
+            <button type="button" class="small_square_button small_square_x" id="clear_status_message_button" >
```

Comment:
The save button should be before the cancel button. The buttons don't work yet, do they? Since I don't see any click handlers.

Reply:
yeah it works when you change the status then you can check it by hovering on your name in user-list